notifications/src/main/java/dk/shape/games/notifications/presentation/viewmodels/notifications/SubjectNotificationSwitcherViewModel.kt

```diff
@@ -26,12 +27,23 @@ internal class SubjectNotificationSwitcherViewModel(
             )
     )
 
-    val item: ObservableField<Any> = ObservableField<Any>().onChange {
-        onItemChanged()
+    val item: ObservableField<Any> = ObservableField<Any>(loadingViewModel).onChange {
+        if (it is SubjectNotificationViewModel.Content) {
+            if (lastViewModel !is SubjectNotificationViewModel.Skeleton) {
+                onItemChanged()
+            }
+        }
+        lastViewModel = it
     }
 
     fun showContent(viewModel: SubjectNotificationViewModel) {
-        item.set(viewModel)
+        if (viewModel is SubjectNotificationViewModel.Skeleton) {
```

Comment:
should this if get combined?
```suggestion
        if (viewModel is SubjectNotificationViewModel.Skeleton && item.get() !is SubjectNotificationViewModel.Content) {
```
what happens if viewModel is SubjectNotificationViewModel.Skeleton but the current item is not content?

Reply:
if the current item is not content we set it otherwise we don't do anything cause we don't want to put the skeleton if the current is already content

Reply:
ok